src/Version.php

```diff
@@ -84,7 +84,7 @@ public function __toString(): string
      *
      * @param string $version Version string
      *
-     * @throws \PHLAK\SemVer\Exceptions\InvalidVersionException
+     * @throws InvalidVersionException if the provided semantic version string is invalid
```

Comment:
```suggestion
     * @throws InvalidVersionException
```

Reply:
I think complete PHPDoc is good practice. You have to decide. 🙃